### DISCUSSION TITLE
Recommended new plugin pathway?

### DISCUSSION THREAD

**Discussion by hpaves** [vkurko/calendar#255] [Q&A] [answered]
Right now I've simply made a copy of one of your plugins and modified it to suit my purposes, but this approach is far from ideal. What would be your recommended pathway of making my own plugin?

**Comment by vkurko** [comment, marked as answer]
You could publish your plugin to [npmjs](https://docs.npmjs.com/packages-and-modules/contributing-packages-to-the-registry) and use it just like the official ones `npm install [your-plugin]`.

**Reply by hpaves** [reply]
Thanks!
